config/rabbitmq.php

```diff
@@ -55,4 +55,15 @@
      */
     'sleep_on_error' => env('RABBITMQ_ERROR_SLEEP', 5),
 
+    /*
+     * Optional SSL params if an SSL connection is used
+     */
+    'ssl_params' => [
+        'ssl_on'        => env('RABBITMQ_SSL', false),
+        'cafile'        => env('SSL_CAFILE', null),
```

Comment:
Always add `RABBITMQ` prefix to all variable names

Reply:
The reason why I didn't do it there was the the same setting can be used for SSL connections to the database and other services. 
So once you start using SSL, the path to the CA file doesn't have to be related just to RabbitMQ.

Reply:
We are not eligible to make changes that can affect other parts of framework. So, this option should be strictly to this library.

Reply:
That makes sense. I'll change it and update the pull request.